### PR TITLE
Git/week1

### PR DESCRIPTION
**Pull request by somayehdvr**

### CODE REVIEW

**Comment by martamatos** [comment]
Very good Somayeh! :D

I have one last task for you, to mimic the process of implementing feedback: can you add another country to the countries.txt file, commit and push please? And then you'll see that the PR gets updated automatically.

File: git/week1/countries.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Iran
+Iraq
+Syria
```
Comment (somayehdvr) [inline git/week1/countries.txt:3]: Saudi Arabia

**Comment by somayehdvr** [comment]
> Very good Somayeh! :D
> 
> I have one last task for you, to mimic the process of implementing feedback: can you add another country to the countries.txt file, commit and push please? And then you'll see that the PR gets updated automatically.

Done! Thanks.

**Review by martamatos — APPROVED** [review]
Great! You can now merge your PR :D

**Pull request merged**